scipy/optimize/_root_scalar.py

```diff
@@ -436,6 +453,29 @@ def _root_scalar_secant_doc():
     """
     pass
 
+def _root_scalar_muller_doc():
+    r"""
+    Options
+    -------
+    args : tuple, optional
+        Extra arguments passed to the objective function.
+    xtol : float, optional
```

Comment:
I'll ask for separate relative and absolute abscissa *and function* tolerance for the elementwise interface. Consider whether you want to include them here.